sdk/core/core-http/src/fetchHttpClient.ts

```diff
@@ -55,6 +55,9 @@ export abstract class FetchHttpClient implements HttpClient {
 
     const abortController = new AbortController();
     let abortListener: ((event: any) => void) | undefined;
+    let registerCleanUpListenerOnStream:
```

Comment:
instead of conditionally defining the method and checking its existence later, can't we just check `if (httpRequest.abortSignal)` instead?